### PR TITLE
Fix sending emails to default_recipient when it's a group

### PR DESCRIPTION
**Pull request by phillipthelen**
E-Mailtemplates can have default recipients their emails are sent to during the automatic email triggers. this was broken in a way that only the first user in the list of groups would get the email. This change fixes it.

the PR also has some small changes to make the project Django 4.0 compatible.

### CODE REVIEW

**Comment by xi** [comment]
This is partly a duplicate of #388. You found more cases though. (I haven't had a closer look).

We really have to find a way to avoid duplicate work. Someone has to take care of reviewing pull requests!

**Comment by phillipthelen** [comment]
@xi Oh, you're right. I completely missed that. I'll try to be more on top of reviewing pull requests again.

I merged your PR in and then fixed the conflicts with this branch, so it should now hopefully be correct

File: Lagerregal/urls.py [hunk 1]
```diff
@@ -163,7 +165,7 @@
 
     path('oauth2/', include('oauth2_provider.urls', namespace='oauth2_provider')),
 
-    path('', include('favicon.urls')),
+    re_path(r'^favicon\.ico$', RedirectView.as_view(url=conf.FAVICON_PATH, permanent=True), name='favicon'),
```
Comment (xi) [inline Lagerregal/urls.py]: No need to use `re_path()` here.

File: devices/views.py [hunk 2]
```diff
@@ -24,7 +24,8 @@
 from django.urls import reverse_lazy
 from django.utils import timezone
 from django.utils.timesince import timesince
-from django.utils.translation import ugettext_lazy as _
+from django.utils.timezone import utc
```
Comment (xi) [inline devices/views.py]: This import is not used

**Review by xi — DISMISSED** [review]
Apart from the two small comments above this looks good to me.

**Review by xi — APPROVED** [review]

**Pull request merged**